File: src/main/scala-graphql/io/github/graphql/model/LicensesQueryResponse.scala
package io.github.graphql.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult
import java.util.{ Map => JMap }

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:01+0800"
)
class LicensesQueryResponse extends GraphQLResult[JMap[String, Seq[LicenseTO]]] {

    def licenses(): Seq[LicenseTO] = {
        val data: JMap[String, Seq[LicenseTO]] = getData
        if (data != null) data.get(LicensesQueryResponse.OPERATION_NAME) else null
    }

}

object LicensesQueryResponse {

    private final val OPERATION_NAME: String = "licenses"

}

File: src/main/java-graphql/io/github/graphql/java/model/LicensesQueryResponse.java
package io.github.graphql.java.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult;
import java.util.Map;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-07T19:46:14+0800"
)
public class LicensesQueryResponse extends GraphQLResult<Map<String, java.util.List<LicenseTO>>> {

    private static final String OPERATION_NAME = "licenses";

    public LicensesQueryResponse() {
    }

    public java.util.List<LicenseTO> licenses() {
        Map<String, java.util.List<LicenseTO>> data = getData();
        return data != null ? data.get(OPERATION_NAME) : null;
    }

}
